bundles/org.openhab.binding.ecotouch/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,1005 @@
+			<channel id="operating_hours_source_pump" typeId="operating_hours_source_pump"/>
+			<channel id="operating_hours_solar" typeId="operating_hours_solar"/>
+			<channel id="enable_heating" typeId="enable_heating"/>
+			<channel id="enable_cooling" typeId="enable_cooling"/>
+			<channel id="enable_warmwater" typeId="enable_warmwater"/>
+			<channel id="enable_pool" typeId="enable_pool"/>
+			<channel id="enable_pv" typeId="enable_pv"/>
+			<channel id="state" typeId="state"/>
+			<channel id="state_sourcepump" typeId="state_sourcepump"/>
+			<channel id="state_heatingpump" typeId="state_heatingpump"/>
+			<channel id="state_evd" typeId="state_evd"/>
+			<channel id="state_compressor1" typeId="state_compressor1"/>
+			<channel id="state_compressor2" typeId="state_compressor2"/>
+			<channel id="state_extheater" typeId="state_extheater"/>
+			<channel id="state_alarm" typeId="state_alarm"/>
+			<channel id="state_cooling" typeId="state_cooling"/>
+			<channel id="state_water" typeId="state_water"/>
+			<channel id="state_pool" typeId="state_pool"/>
+			<channel id="state_solar" typeId="state_solar"/>
+			<channel id="state_cooling4way" typeId="state_cooling4way"/>
+			<channel id="alarm" typeId="alarm"/>
+			<channel id="interruptions" typeId="interruptions"/>
+			<channel id="state_service" typeId="state_service"/>
+			<channel id="adapt_heating" typeId="adapt_heating"/>
+			<channel id="manual_heatingpump" typeId="manual_heatingpump"/>
+			<channel id="manual_sourcepump" typeId="manual_sourcepump"/>
+			<channel id="manual_solarpump1" typeId="manual_solarpump1"/>
+			<channel id="manual_solarpump2" typeId="manual_solarpump2"/>
+			<channel id="manual_tankpump" typeId="manual_tankpump"/>
+			<channel id="manual_valve" typeId="manual_valve"/>
+			<channel id="manual_poolvalve" typeId="manual_poolvalve"/>
+			<channel id="manual_coolvalve" typeId="manual_coolvalve"/>
+			<channel id="manual_4wayvalve" typeId="manual_4wayvalve"/>
+			<channel id="manual_multiext" typeId="manual_multiext"/>
+
+			<channel id="temperature_surrounding" typeId="temperature_surrounding"/>
+			<channel id="temperature_suction_air" typeId="temperature_suction_air"/>
+			<channel id="temperature_sump" typeId="temperature_sump"/>
+
+			<channel id="ecovent_temp_exhaust_air" typeId="ecovent_temp_exhaust_air"/>
+			<channel id="ecovent_temp_exit_air" typeId="ecovent_temp_exit_air"/>
+			<channel id="ecovent_temp_outdoor_air" typeId="ecovent_temp_outdoor_air"/>
+			<channel id="ecovent_temp_supply_air" typeId="ecovent_temp_supply_air"/>
+			<channel id="ecovent_CO2_value" typeId="ecovent_CO2_value"/>
+			<channel id="ecovent_moisture_value" typeId="ecovent_moisture_value"/>
+			<channel id="ecovent_output_y1" typeId="ecovent_output_y1"/>
+			<channel id="ecovent_mode" typeId="ecovent_mode"/>
+		</channels>
+
+		<config-description>
+			<parameter name="ip" type="text" required="true">
+				<label>IP address / hostname of the heat pump</label>
```

Comment:
```suggestion
			<parameter name="ip" type="text" required="true">
			<context>network-address</context>
				<label>Address of Heat Pump</label>
```
Description gives more info and the context will/may get used to ensure a valid entry is made.

Reply:
didn't know about this. ok.